lib/spoom/minitest/backtrace_filter.rb

```diff
@@ -0,0 +1,21 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "minitest"
+
+module Spoom
+  module BacktraceFilter
+    class Minitest < ::Minitest::BacktraceFilter
```

Comment:
I originally had `Spoom::Minitest::BacktraceFilter` but that causes typecheck problems with tests in the `Spoom` namespace that inherit from `Minitest::Test`.

Reply:
I think this way is ok, have renamed the file to match.